hologram_dashboard/fetcher/hologram_config_malformed.config

```diff
@@ -0,0 +1,63 @@
+data_source_config {
```

Comment:
Move this to hologram_dashboard/fetcher/testdata/

Reply:
+1. And make this file short, with some fake value.